build.gradle

```diff
@@ -1,7 +1,6 @@
 group 'com.nitishm.userlist'
```

Comment:
your group and the package declarations dont match.
either have this as `org.nitishm.javacollectionassignment`
or have the source code packages as `com.nitishm.userlist`